shared-libs/transitions/src/transitions/index.js

```diff
@@ -148,17 +149,19 @@ const loadTransitions = (synchronous = false) => {
     try {
       self._loadTransition(transition, synchronous);
     } catch (e) {
-      loadError = true;
-      logger.error(`Failed loading transition "${transition}"`);
+      const errorMessage = `Failed loading transition "${transition}"`;
+      loadError = [errorMessage, e && e.message || 'unknown'];
```

Comment:
Another nit: this ends up being logged as an array, again with the quotes getting escaped: 
```
2023-04-06 13:15:52 ERROR: Transitions are disabled: ["Failed loading transition \"death_reporting\"","Configuration error. Config must have a 'death_reporting.mark_deceased_forms' array defined."] 
```